app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -78,8 +78,8 @@ fun WallpaperSettings(
             .verticalScroll(rememberScrollState())
             .background(color = FirefoxTheme.colors.layer1)
             .padding(
-                end = 16.dp,
-                start = 16.dp,
+                end = 12.dp,
+                start = 12.dp,
```

Comment:
Is this intentional? This is still 16dp in https://www.figma.com/file/d7l1iGabQYqnLcnQbHDGzM/Wallpaper-2.0-%2B-Onboarding-v106?node-id=5943%3A203265

Reply:
I believe this was a result of the UX design review. It looks like there is additional padding added to each element here:
```
val wallpaper = wallpapers[itemIndex]
                    Box(
                        modifier = Modifier
                            .weight(1f, fill = true)
                            .padding(4.dp),
                    ) 
```